### DISCUSSION TITLE
Need Advice

### DISCUSSION THREAD

**Discussion by Ashedamahraomai** [community/community#113332] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

What is the best CSS framework to use with React, I am just starting out with React so would like some suggestion.

**Comment by deka27** [comment]
It depends to be honest.

Tailwind, Bootstrap, MUI, chakra are all good. Personal Preference would be Tailiwind CSS. It takes some time to get used to though, but later it does the job for you. Again, it all depends on the preference, as you are just starting out, go with something you can get comfortable with easily.

Here is a reddit post for the same [Link](https://www.reddit.com/r/react/comments/10gi3n0/what_is_the_best_css_framework_to_use_with_react/)

**Comment by ettaboyle** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
